server/cli/launch.py

```diff
@@ -143,6 +149,13 @@ def launch(
 
     if not port:
         port = find_available_port(host)
+    else:
```

Comment:
if you are feeling like it, you could flip the if else so that it's a positive `if` and not an `if not`